lib/gossip/node.ex

```diff
@@ -21,26 +26,116 @@ defmodule Gossip.Node do
   # Server (callbacks)
 
   @impl true
-  def init(:ok) do
+  def init(opts) do
+    sum = Enum.at(opts, 0)
```

Comment:
Can you pass opts as Keyword list? 

Refer over here -> https://hexdocs.pm/elixir/Keyword.html

Would be better to know which `opts` we are actually parsing